Header            header
sensor_msgs/Image image
int16[] bounding_box
float32           certainty
geometry_msgs/Point[] shape
string            face_id
